Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,360 @@
+            response = self._http_request(method="GET", url_suffix='/tweets/search/recent',
+                                          headers=self._headers, params=query_params,
+                                          ok_codes=[200])
+            result, next_token = create_context_data_search_tweets(response)
+        except Exception as e:
+            raise e
+        return response, result, next_token
+
+    def twitter_user_get(self, users_names: list[str], return_pinned_tweets: str) -> tuple[dict, list[dict]]:
+        """ Gets users according to the provided user names.
+            Args:
+                user_name: list[str] - List of users names.
+                return_pinned_tweets: str - Indicates whether to return user's pinned Tweets.
+                limit: int - Maximum number of results to return.
+
+            Returns:
+                A tuple[dict, dict, Optional[str]] with:
+                dict: raw response.
+                dict: context data.
+        """
+        result = []
+        query_params = {'usernames': ','.join(f'{item}' for item in users_names),
+                        'user.fields': 'created_at,description,entities,id,location,name,pinned_tweet_id,profile_image_url'
+                        ',protected,public_metrics,url,username,verified,withheld'}
+        if return_pinned_tweets == 'true':
+            query_params['expansions'] = 'pinned_tweet_id'
+            query_params['tweet.fields'] = 'id,text,attachments,conversation_id,created_at,public_metrics'
+        try:
+            response = self._http_request(method="GET", url_suffix='/users/by',
+                                          headers=self._headers, params=query_params,
+                                          ok_codes=[200])
+            result = create_context_data_get_user(response, return_pinned_tweets)
+        except Exception as e:
+            raise e
+        return response, result
+
+
+''' HELPER FUNCTIONS '''
+
+
+def create_context_data_search_tweets(response: dict) -> tuple[List[dict], str]:
+    """ Gets raw response form Twitter API and extracts the relevent data.
+        The data matched by
+        attachments.media_keys == includes.media.media_key
+        and author_id == includes.users.id.
+            Args:
+                response: dict - raw response form Twitter API.
+            Returns:
+                A tuple[dict, str] with:
+                dict: context data.
+                str: next token.
+    """
+    include = response.get('includes', {})
+    data = response.get('data', [])
+    users = include.get('users', [])
+    media = include.get('media', [])
+    next_token = response.get('meta', {}).get('next_token')
+    list_dict_response = []
+    for data_item in data:
+        author_id = data_item.get('author_id')
+        for user in users:
+            id = user.get('id')
+            if author_id == id:
+                dict_to_append = {
+                    'id': data_item.get('id', None),
```

Comment:
Same here, why all the Nones?

Reply:
This enables to the function remove_empty_elements to remove it, if it is empty.

Reply:
You don't need all the Nones
The get itself will return None if no value is found
https://www.w3schools.com/python/ref_dictionary_get.asp